score.js

```diff
@@ -0,0 +1,38 @@
+class Score {
+  constructor() {
+    this.frames = [];
+  };
+  
+  add(frame) {
+    this.frames.push(frame);
+  };
+
+  forFrame(number) {
+    if (this.frames[number - 1].isStrike()) {
+      if (this.frames[number].isStrike()) {
+        return 20 + this.frames[number + 1].roll_1
+      }
+      
+      return 10 + this.frames[number].getSum();
```

Comment:
I like the bonus calculating using the previous frame, I encountered a lot of difficulty with the reverse i.e using the next frame